Assets/GothicVR/Scripts/UI/TurnSettingDropdownController.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+using GVR.Phoenix.Interface;
+public class TurnSettingDropdownController : MonoBehaviour
+{
+    public GameObject textAsset;
```

Comment:
My IDE tells me it's unused. Can it be removed?

![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/151c42dc-2452-46f6-8a2b-c0bb7b674386)

Reply:
Was a legacy implementation. Removed.